### PR TITLE
[TEVA-2014] remove required/add optional to application process

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-2014

there will be more PRs for this ticket but breaking it down into manageable chucks.

this uses new approach i put in engineering channel earlier - this will be applied to what i did yesterday for create vacancy. nothing changes from the UI look perspective just using CSS/templates to avoid knarly repetitive HTML in locale files

### CODE REVIEW

File: app/views/jobseekers/job_applications/build/personal_details.html.slim [hunk 1]
```diff
@@ -10,17 +10,17 @@
 
     = form_for @form, url: wizard_path, method: :patch do |f|
       = f.govuk_error_summary
-      = f.govuk_text_field :first_name, label: { size: "s" }, width: "one-half"
-      = f.govuk_text_field :last_name, label: { size: "s" }, width: "one-half"
-      = f.govuk_text_field :previous_names, label: { size: "s" }, width: "one-half"
-      = f.govuk_fieldset legend: { text: t("helpers.label.jobseekers_job_application_personal_details_form.your_address_html") } do
+      = f.govuk_text_field :first_name, label: { size: "s" }, width: "one-half", aria: { required: true }
```
Comment (david-mears-dfe) [inline app/views/jobseekers/job_applications/build/personal_details.html.slim:13]: 👍🏻

**Review by cesidio — APPROVED** [review]

File: config/locales/forms.yml [hunk 2]
```diff
@@ -220,18 +220,18 @@ en:
...
+        last_name: Last name
+        national_insurance_number: National Insurance number
+        phone_number: Phone number
         previous_names: Previous names
         town_or_city: Town or city
-        teacher_reference_number_html: Teacher reference number (<span class='text-red'>Required</span>)
+        teacher_reference_number: Teacher reference number
       jobseekers_job_application_personal_statement_form:
-        personal_statement_html: Your personal statement (<span class='text-red'>Required</span>)
+        personal_statement_html: Your personal statement
```
Comment (david-mears-dfe) [inline config/locales/forms.yml:234]: minus html

**Review by david-mears-dfe — APPROVED** [review]
Approved with minor comment

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2844.london.cloudapps.digital>

**Review by cpjmcquillan — APPROVED** [review]

**Pull request merged**